hutch_python/load_conf.py

```diff
@@ -293,8 +322,40 @@ def load_conf(conf, hutch_dir=None):
 
     # Daq
     with safe_load('daq'):
-        cache(daq=Daq(RE=RE, hutch_name=hutch))
-        cache.doc(daq='LCLS1 DAQ interface object.')
+        if daq_type.startswith('lcls1'):
+            if daq_type == 'lcls1-sim':
```

Comment:
```suggestion
            if daq_type == 'lcls1-sim' or (args and args.sim):
```
Maybe?

Reply:
Hmm, seems good- maybe combined with removing the other place where the code sets the tim

Reply:
Cleans it up to avoid setting the sim in two places